invenio_rdm_records/services/pids/manager.py

```diff
@@ -181,7 +181,7 @@ def create_all(self, draft, pids=None, schemes=None):
             result[scheme] = self.create(
                 draft,
                 scheme,
-                pid_attrs["identifier"],
+                pid_attrs.get("identifier", None),
```

Comment:
are we sure that direct key access wasn't there on purpose to fail early?
minor: adding None as get function argument is redundant since it is default